### DISCUSSION TITLE
如何让子节点使用自定义端口

### DISCUSSION THREAD

**Discussion by kettly1260** [EasyTier/EasyTier#508] [Q&A]
使用的NAT机器，发现需要端口没办法直接放行，对应的接口不一致，或者对等接口是随机分配的，能否添加自定义接口，但可以同时接入公共服务器呢

**Comment by rainpaper-bs** [comment]
你的接口是端口的意思吗？对于不对称的端口映射，新版有一个功能，在官网文档 https://easytier.cn/guide/network/p2p-optimize.html 里的”指定公网 IP 和端口“章节，你看下，可以手动指定
